### DISCUSSION TITLE
What's the difference between `kwargs_row` and `class_row`?

### DISCUSSION THREAD

**Discussion by Kludex** [psycopg/psycopg#704] [Q&A]
I see the implementation is the same: https://github.com/psycopg/psycopg/blob/ec7a6adfc0b5ffd8d8fb7e247636ab89589c5c0b/psycopg/psycopg/rows.py#L155-L211

But on our internal code, there was a difference in behavior, which we couldn't identify why... For more context, we were using only `kwargs_row` for both callables and classes.

Is there some internal logic that does something different on them?

cc @adriangb

**Comment by dvarrazzo** [comment]
Practically, none :innocent:
